beacon-chain/core/peerdas/helpers.go

```diff
@@ -43,15 +43,15 @@ var (
 	maxUint256 = &uint256.Int{math.MaxUint64, math.MaxUint64, math.MaxUint64, math.MaxUint64}
 )
 
-// CustodyColumns computes the columns the node should custody.
-// https://github.com/ethereum/consensus-specs/blob/dev/specs/_features/eip7594/das-core.md#helper-functions
-func CustodyColumns(nodeId enode.ID, custodySubnetCount uint64) (map[uint64]bool, error) {
+// GetCustodyColumns computes the columns the node should custody.
```

Comment:
I don't personally feel strongly about it, but "Get" is typically omitted in Go for idiomatic getters: https://go.dev/doc/effective_go

Reply:
That's fair. It's up to you guys. I think it makes most sense to do that style when the object is a "record" and when the function doesn't perform some computation.

Reply:
I've thought about it some more. I like the idea of being Go-idiomatic. Will close this PR.